cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -19,9 +19,9 @@ To create a full-fledged, self-serving CI/CD pipeline for an application, you mu
 
 This section uses the `pipelines-tutorial` example to demonstrate the preceding tasks. The example uses a simple application which consists of:
 
-* A front-end interface, `vote-ui`, with the source code in the link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-2[`ui-repo`] Git repository.
-* A back-end interface, `vote-api`, with the source code in the link:https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-2[`api-repo`]  Git repository.
-* The `apply-manifests` and `update-deployment` tasks in the link:https://github.com/openshift/pipelines-tutorial/tree/release-tech-preview-2[`pipelines-tutorial`] Git repository.
+* A front-end interface, `vote-ui`, with the source code in the link:https://github.com/openshift-pipelines/vote-ui/tree/pipelines-1.3[`ui-repo`] Git repository.
+* A back-end interface, `vote-api`, with the source code in the link:https://github.com/openshift-pipelines/vote-api/tree/pipelines-1.3[`api-repo`]  Git repository.
+* The `apply-manifests` and `update-deployment` tasks in the link:https://github.com/openshift/pipelines-tutorial/tree/pipelines-1.3[`pipelines-tutorial`] Git repository.
 
 == Prerequisites
 
```

Comment:
@Preeticp under `Prerequisites` i see 
```
* You have forked the front-end link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-2[`ui-repo`] and back-end link:https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-2[`api-repo`] Git repositories using your GitHub ID, and have Administrator access to these repositories.
```
pointing to wrong branch which is `release-tech-preview-2`

Reply:
Thank you so much Savita!!